### ISSUE TITLE
Absolute positioned image  with alt text not loading on Chrome 92

### ISSUE THREAD

**Issue by makebit** [verlok/vanilla-lazyload#532]
**Describe the bug**
An image with absolute positioning is not loaded when it has an alt text tag.
The issue is present only in the latest version of Chrome (92)

**To Reproduce**
See:
[https://codepen.io/makebit/pen/gOWBoYM](https://codepen.io/makebit/pen/gOWBoYM)
with Chrome 92

**Expected behavior**
The image should be loaded with and without the alt text tag

**LazyLoad version**
Please report which version of LazyLoad you're using.
- Version 17.4.0

**Desktop (please complete the following information):**
 - OS: any
 - Browser: Chrome
 - Version: 92

**Smartphone (please complete the following information):**
 - Device: any
 - OS: any
 - Browser: Chrome
 - Version: 92

**Additional context**
It works flawless with Firefox.

I hope you can help me! Thank you

**Comment by makebit** [comment]
Setting ```use_native``` to ```true``` seems to fix the problem. However it is not clear to me why this happens.

**Comment by verlok** [comment]
Hey there, 
sorry for being this late, crazy summer here.

I've checked your Codepen, [forked it](https://codepen.io/verlok/pen/eYRWXMZ), and found that your images are positioned `absolute` and `9999px` out of their container.
That might be the issue that confuses Chrome's rendering engine.

If you inspect the page, you'll see that the first image isn't inside the viewport at all.
Whereas if you remove the `alt` attribute, the image appears inside again.

Additionally, if you remove the following CSS, everything work as expected.

```css
.news-3 .news-img img {
    position: absolute;  
    top: -9999px;
    bottom: -9999px;
    left: -9999px;
    right: -9999px;
    margin: auto;
}
```

So this this bug should be addressed to the [Chromium bugs](https://bugs.chromium.org/) and not to this library, because the intersection is calculated by the browser using the IntesectionObserver API.

**Comment by verlok** [comment]
@makebit any updates? Any luck dolci this issue?

**Comment by verlok** [comment]
Hey @makebit , 
I’m going to assume you solved so I'm closing this issue.

If you’re happy with the documentation and the effort I’ve been putting on this project in the latest years, I hope you might want to [buy me a coffee](https://ko-fi.com/verlok) to show your appreciation. Or [sponsor me](https://github.com/sponsors/verlok), if you're a fan.

Open-source software is great for everyone, but it takes passion and time to grow and evolve.

Thank you for thinking about it. 
Have a wonderful day!
